### ISSUE TITLE
Colorpicker accessibility issues

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1598]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. It doesn't announce itself as a colorpicker, just a table.
Expected: Should say "Colorpicker, use arrow keys to navigate to a color textbox control to change color"

2. Text boxes are not labeled, just say their roles.

**Comment by betrue-final-final** [comment]
All this still repros in Narrator.

**Comment by betrue-final-final** [comment]
Still valid.
